### DISCUSSION TITLE
wl_output issue

### DISCUSSION THREAD

**Discussion by potter-jason** [any1/wayvnc#167] [Q&A]
Hello
On a clean install of Fedora 37 Workstation I'm getting the following error:
[feduser@fedora ~]$ wayvnc 0.0.0.0
wl_registry@2: error 0: invalid version for global wl_output (4): have 2, wanted 3
ERROR: ../src/main.c: 363: Virtual Pointer protocol not supported by compositor.
ERROR: ../src/main.c: 364: wayvnc may still work if started with --disable-input.
ERROR: ../src/main.c: 953: Failed to initialise wayland

What dependency am I missing that I need to install? thanks!

**Comment by any1** [comment]
It looks like you're not running a wlroots based compositor.
